cmd/flux/fmt.go

```diff
@@ -87,7 +86,7 @@ func format(script string) (bool, error) {
 }
 
 func updateScript(fname string, script string) error {
-	err := ioutil.WriteFile(fname, []byte(script+"\n"), 0644)
+	err := ioutil.WriteFile(fname, []byte(script), 0644)
```

Comment:
This is because it's done by the rust formatter now, right?

Reply:
Yup

Reply:
So we did this correctly via the command line, just not in the LSP which only called into format. Since this was appended after we did the check to see if the formatting changed we didn't end up adding this newline to a script which did lack a newline unless the formatting changed for other reasons. So we ended up having a few files that without this newline also